src/components/Footer/Footer.scss

```diff
@@ -0,0 +1,53 @@
+.footer {
+  color: #777;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)